src/components/CompareResults/beta/RevisionSelect.tsx

```diff
@@ -41,6 +54,8 @@ function RevisionSelect() {
             </Box>
           );
         }}
+        onChange={(event) => onChangeHandler(event)}
```

Comment:
(optional) I think you can simplify by reducing the number of functions we need to find and read to understand what's happening:
```suggestion
        onChange={(event) => handlerChangeComparison(event.target.value)}
```
(possibly with `void` I'm not sure)